recipes/rsirius/test_script.R

```diff
@@ -0,0 +1,23 @@
+options(error = traceback)
+
+library('Rsirius')
+
+sdk <- SiriusSDK$new()
+wrapper <- sdk$start()
+
+wait_for_job <- function(pid, job) {
+  while (!(wrapper$computations_api$GetJob(pid, job$id)$progress$state == "DONE")) {
+    Sys.sleep(1)
+  }
+}
+
+pspace <- wrapper$project_spaces_api$GetProjectSpaces()[[1]]$name
+data <- file.path(Sys.getenv('RECIPE_DIR'),"Kaempferol.ms")
```

Comment:
This workaround works, but it hides the conda aspect (the fact that `RECIPE_DIR` is defined by conda-build). The more explicit method is to list it as a [test file](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-files) in the test section so that it is copied to the test directory:

```yaml
test:
  files:
    - Kaempferol.ms
```